scripts/github-workflow-e2e-matrix.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnSync } from 'child_process';
+
+try {
+  const all = process.argv.includes('--all');
+  const affectedJson = runCommand(
+    'npx',
+    'nx',
+    'print-affected',
+    '--target=build-storybook',
+    all ? ' --all' : ''
+  );
+  const affected = JSON.parse(affectedJson);
+  const projects = affected.tasks
+    .map((task: any) => task.target.project)
+    .filter((project: string) => project && project !== 'storybook');
```

Comment:
Could you use `--exclude=storybook` in the CLI command instead? This will make it easier to include other excluded projects in the future.

Reply:
I switched to `--exclude=storybook`, and this code has moved to the `dev-infra` repo.